### DISCUSSION TITLE
Welcome to UXF Discussions!

### DISCUSSION THREAD

**Discussion by jackbrookes** [immersivecognition/unity-experiment-framework#56] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by muhammedsa** [comment]
Hello guys,

I'm interested in contributing to this project by reporting and documenting issues.

I'm new to open-source and VR so I just want to know where I can start.

**Reply by jackbrookes** [reply]
Hi! For specific issues you can go to the "Issues" tab and create a new topic. Issues can be feature suggestions, bugs, reporting missing documentation, or just asking for help or ideas! There is no strict format for issues, but try to list a series of step to reproduce the bug, if that is what you have an issue with.
